### DISCUSSION TITLE
v1.2.1

### DISCUSSION THREAD

**Discussion by lucacome** [lucacome/docker-image-update-checker#24] [Announcements]
### 🐛 Bug Fixes
* Fix retrieving manifest for single platform by @lucacome in https://github.com/lucacome/docker-image-update-checker/pull/23
### 🔨 Maintenance
* Enable trace based on debug flag by @lucacome in https://github.com/lucacome/docker-image-update-checker/pull/21

**Full Changelog**: https://github.com/lucacome/docker-image-update-checker/compare/v1.2.0...v1.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lucacome/docker-image-update-checker/releases/tag/v1.2.1'>v1.2.1</a>.</em>

**Comment by idydiallo** [comment]
Is there a action for checking if a base image needs updating within a private ecr repo?

**Reply by lucacome** [reply]
@idydiallo it’s supported since https://github.com/lucacome/docker-image-update-checker/releases/tag/v3.1.0
